src/cljc/athens/components_util.cljc

```diff
@@ -0,0 +1,22 @@
+(ns athens.components-util
```

Comment:
I would actually put all your component-related work into one namespace for now: `src/cljs/athens/components.cljs`

- components_util
- website_embeds
- todo

Reply:
Yes, this would've made more senes. Done!